docs/client/data.js

```diff
@@ -5716,7 +5716,17 @@ DocsData = {
     "memberof": "Mongo.Cursor",
     "name": "count",
     "options": [],
-    "params": [],
+    "params": [
+      {
+        "description": "<p>Optional. Specifies whether to consider the effects of the <code>skip</code> and <code>limit</code> options. By default, the <code>count()</code> method ignores the effects of the <code>skip</code> and <code>limit</code> options. Set <code>applySkipLimit</code> to true to consider the effect of these options.</p>",
```

Comment:
This isn't quite accurate, since skip and limit are considered when running on the client always -- this function is defined on the client and on the server, and documented as such.

Not sure off the top of my head what the right way to explain this here is.